docs/developer-docs/integrations/sns/tokenomics/sns-checklist.md

```diff
@@ -13,11 +13,12 @@ launch – that is a decision by the NNS DAO. Ultimately, following this guide a
 an SNS is done at your own risk.
 :::
 
-<!-- NEW: [INFO BOX STYLE] 
+:::info
 The NNS has approved a new flow for how SNSs can be launched that only require one proposal.
 The old flow required two NNS and one SNS proposals.
```

Comment:
```suggestion
The old flow required two NNS and one SNS proposal.```

Reply:
I assume this should just about using singular instead of plural. I ignored the ```